tools/scoring/tests/fixtures/files/proposed/building_config_example.yaml

```diff
@@ -0,0 +1,180 @@
+# Copyright 2022 Google LLC
```

Comment:
can you please put the files in 
```
tests/samples/proposed/
tests/samples/solution/
```

Just to be compatible with the other tools.